### ISSUE TITLE
Deletion URL with confirmation

### ISSUE THREAD

**Issue by SirYwell** [IntellectualSites/SchematicWeb#1]
Currently, using the deletion link generated by SchematicWeb directly deletes the file. That might be unexpected for users, therefore I'm suggesting to add a confirmation dialog where the user needs to explicitely click a button to delete a file.

**Comment by simpleauthority** [comment]
Yeah, I can see the benefit, and I can do it. I still wonder though if the specific plugin shouldn't be the one confirming?

**Comment by SpecifiedNull** [comment]
I think add a tag that can auto confirm deletion upon opening the link for example "https://sw.jacobandersen.dev/delete/c7e061c4fd95456abcde6e06b45eef810?confirm=true" if this tag was not at the end of the url it will prompt user to confirm deletion.
